### PR TITLE
docs/node: Bump Sapphire Testnet to 0.1.4-testnet

### PR DESCRIPTION
**Pull request by kostko**

### CODE REVIEW

File: docs/node/testnet/README.md [hunk 1]
```diff
@@ -86,9 +86,9 @@ Feel free to use other IAS proxies besides the one provided here or [run your ow
 * Runtime identifier:
   * `000000000000000000000000000000000000000000000000a6d1e3ebf60dff6c`
 * Runtime binary version:
-  * [0.1.3-testnet](https://github.com/oasisprotocol/sapphire-paratime/releases/tag/v0.1.3-testnet)
+  * [0.1.4-testnet](https://github.com/oasisprotocol/sapphire-paratime/releases/tag/v0.1.4-testnet)
 * Web3 Gateway version:
-  * [3.0.0](https://github.com/oasisprotocol/emerald-web3-gateway/releases/tag/v3.0.0)
+  * [3.1.0-rc1](https://github.com/oasisprotocol/emerald-web3-gateway/releases/tag/v3.1.0-rc1)
```
Comment (matevz) [inline docs/node/testnet/README.md:91]: I suppose you can also bump Web3 gateway for Emerald above to 3.1.0-rc1?

Comment (kostko) [reply docs/node/testnet/README.md:91]: But the only changes are related to Sapphire.

Comment (kostko) [reply docs/node/testnet/README.md:91]: Oh I see it also specifies an old rc version. Ok will bump.

Comment (kostko) [reply docs/node/testnet/README.md:91]: Done.

**Review by matevz — APPROVED** [review]

**Pull request merged**